tests/app/model_test.py

```diff
@@ -61,13 +61,37 @@ def test_core_data_model(app):
         # doing this crazy thing because the offset between UTC and EST varies depending on the date
         hour_in_et = now_utc.astimezone(pytz.timezone('US/Eastern')).hour
         assert core_data_row.lastUpdateEt == '5/4/2020 %d:03' % hour_in_et
-        
+
         # check that the Batch object is attached to this CoreData object
         assert core_data_row.batch == batch
         # also check the relationship in the other direction, that CoreData is attached to the batch
         assert len(batch.coreData) == 1
         assert batch.coreData[0] == core_data_row
 
+def test_core_data_fields():
+    '''This test tests the valid_fields_checker method in CoreData '''
+
+    # There are some consts and assumptions here about what is a field and what isn't.
+    # It's not read dynamically
+
+    valid_fields = ['positive', 'negative']
+    keys = ['state', 'date', 'batchId']
+    unknown_fields = ['moonBaze', 'marsBase', 'kuiperBeltShield']
+
+    valids, unknowns = CoreData.valid_fields_checker(valid_fields + keys + unknown_fields)
+    assert set(valid_fields) == set(valids)
+    assert set(unknown_fields) == set(unknowns)
+
```

Comment:
extra newline?

Reply:
I donno, there's no linter. I'm used to 2 lines between methods not in a class.
Maybe I'll add flake config and add flake8 to to GH actions checker

Reply:
me too on the 2 lines between methods, this one is within the method though